.github/workflows/test-long.yml

```diff
@@ -15,7 +15,7 @@ concurrency:
 
 jobs:
   test-integration:
-    runs-on: ubuntu-latest
+    runs-on: cosmos-sdk-runner
```

Comment:
Isn't it just `ubuntu-latest-8-cores` or something like that? https://docs.github.com/en/actions/using-github-hosted-runners/controlling-access-to-larger-runners#default-group-for-larger-runners

Reply:
its suppose to be 1, this is an extra runner